themes/origins/sections/multi-column.liquid

```diff
@@ -0,0 +1,162 @@
+{{ 'column-card.css' | asset_url | stylesheet_tag }}
+{{ 'section-multi-column.css' | asset_url | stylesheet_tag }}
+
+<yc-section
+  style="
+    --section-space-top: {{ section.settings.space_top }}px;
+    --section-space-bottom: {{ section.settings.space_bottom }}px;
+    --section-bg-color: {{ section.settings.background_color.hex }};
+    --section-text-color: {{ section.settings.text_color.red }}, {{ section.settings.text_color.green }}, {{ section.settings.text_color.blue }};
+  "
+>
+  {% if section.settings.title or section.settings.description %}
```

Comment:
If I only have description provided the h2 tag would still be included in the DOM
<img width="631" alt="image" src="https://github.com/user-attachments/assets/37363baf-3772-4762-b4cc-bb89ddc348d9" />

Better to include if conditions individually on each of the title and desc.